.travis.yml

```diff
@@ -0,0 +1,60 @@
+version: ~> 1.0
+language: generic
+group: travis_latest
+
+# Only build for develop
+branches:
+  only:
+    - develop
+
+matrix:
+  fast_finish: true
+  include:
+    - python: 3.8
+      name: "Power PC"
+      os: linux
+      arch: ppc64le
```

Comment:
We can probably add s390x here for the sake of #1389, but is IBM Z really that prevalent? I guess the z15 was released in  2019, but I can't think of a single cluster that uses it? (definitely nothing in the top500 list)